modules/system/console/OctoberPasswd.php

```diff
@@ -0,0 +1,129 @@
+<?php namespace System\Console;
+
+use Str;
+use Backend\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * Console command to change the password of a Backend user via CLI.
+ *
+ * This command may only be run through CLI.
+ *
+ * @package october\system
+ */
+class OctoberPasswd extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'october:passwd';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Change the password of a Backend user.';
+
+    /**
+     * Was the password automatically generated?
+     *
+     * @var bool
```

Comment:
```suggestion
     * @var bool Was the password automatically generated
```

For class properties I prefer the shorter docblock format, same goes for the above properties as well

Reply:
Sounds good